### DISCUSSION TITLE
Self-Signed cert invalid in firefox

### DISCUSSION THREAD

**Discussion by ackstorm23** [TechnitiumSoftware/DnsServer#733] [Q&A] [answered]
I have SSL turned on, with no cert file so that it generates it's own cert.

However, firefox does not tolerate the cert.

"SEC_ERROR_INADEQUATE_KEY_USAGE"

**Comment by ackstorm23** [comment]
Is this a problem on my side somehow, or should I file a bug report?

**Reply by ShreyasZare** [reply, marked as answer]
Thanks for asking. This error can mean that the Firefox's cert store file is corrupt. You can fix it as explained [here](https://support.mozilla.org/en-US/questions/1310266).
